dl1_data_handler/reader.py

```diff
@@ -168,7 +168,7 @@ def __init__(self,
         if image_channels is None:
             image_channels = ['charge']
         self.image_channels = image_channels
-        self.image_mapper = ImageMapper(mapping_method, image_channels,
+        self.image_mapper = ImageMapper(channels=self.image_channels,
```

Comment:
Raises a KeyError if mapping_settings is None (cause defined as an empty dict)

Reply:
The mapping_method is still needed, but I moved it to the mapping_settings dictionary since there's no reason it has to be separate.